Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -114,14 +147,22 @@ ProjectPamphletViewModelOutputs {
     self.viewWillAppearAnimated.value = animated
   }
 
+  fileprivate let willTransitionToCollectionProperty =
+    MutableProperty<UITraitCollection>(UITraitCollection())
```

Comment:
The type here should be optional and instantiated with `nil`, then in the signal where we use it `skipNil()`. Alternatively we can consider using a `pipe`. I started doing this to avoid the `skipNil()`'s, I'll slack you an example.

Reply:
For now I will use the `skipNil()` because all the other functions on the file uses this pattern. But will consider to use the `pipe()` alternative on future projects.